ReactCommon/react/renderer/components/view/YogaLayoutableShadowNode.cpp

```diff
@@ -751,6 +751,26 @@ void YogaLayoutableShadowNode::swapLeftAndRightInViewProps(
   auto &props = const_cast<ViewProps &>(typedCasting);
 
   // Swap border node values, borderRadii, borderColors and borderStyles.
+  if (props.borderRadii.startEnd.has_value()) {
+    props.borderRadii.topEnd = props.borderRadii.startEnd;
+    props.borderRadii.startEnd.reset();
+  }
+
+  if (props.borderRadii.startStart.has_value()) {
+    props.borderRadii.topStart = props.borderRadii.startStart;
+    props.borderRadii.startStart.reset();
+  }
+
+  if (props.borderRadii.endEnd.has_value()) {
+    props.borderRadii.bottomEnd = props.borderRadii.endEnd;
+    props.borderRadii.endEnd.reset();
+  }
+
+  if (props.borderRadii.endStart.has_value()) {
+    props.borderRadii.bottomStart = props.borderRadii.endStart;
+    props.borderRadii.endStart.reset();
+  }
+
```

Comment:
Looking at the examples below, they are always changing a left-or-right cardinal direction to a flow-relative one. E.g. nothing is done to `borderRadii.topStart`. But the added code seems to be doing the opposite? E.g. changing start/end to a top-bottom direction.

Are you sure the added code here is needed?